python/Cargo.toml

```diff
@@ -47,6 +47,7 @@ lance-io = { path = "../rust/lance-io" }
 lance-linalg = { path = "../rust/lance-linalg" }
 lance-table = { path = "../rust/lance-table" }
 lance-datafusion = { path = "../rust/lance-datafusion" }
+libc = "0.2.176"
```

Comment:
Is this going to be a problem on Windows?

Reply:
Based on what I've found online, it is possible to link against the `libc` crate on Windows.